app/views/administrateurs/procedures/index.html.haml

```diff
@@ -1,6 +1,8 @@
 .sub-header
-  .procedure-admin-listing-container
-    = link_to "Nouvelle Démarche", new_from_existing_admin_procedures_path, id: 'new-procedure', class: 'fr-btn'
+  .flex.fr-container
+    %h1.fr-h3 Mes démarches
+    .procedure-admin-listing-container
```

Comment:
```suggestion
    .procedure-admin-listing-container.fr-mt-1v
```

ça équivaut à 4px. mais au moins on garde une homogénéité d'espacement au travers des spacers du dsfr